Marlin/src/feature/pause.cpp

```diff
@@ -168,7 +170,7 @@ bool pause_print(const float &retract, const float &z_lift, const float &x_pos,
     }
 
     // Unload filament
-    set_destination_to_current();
+    set_destination_to_current();// apply actual position value after move
```

Comment:
Now , a RUNPLAN just after a doblockmove , then we set current and destination ' already in the code ' but just for commenting

Question : I love doblockmove , can you make it with ' extrusion argument ' please 😄 the code will be very faster and clean